src/common/utils/openBrowser.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { shell } from "electron";
+
+const allowedProtocols =  new Set(["http:", "https:"]);
+
+/**
+ * Opens a link using the program configured as the default browser
+ * on the target platform. Will reject URLs with a scheme other than
+ * http or https to prevent programs other than the default browser
+ * running.
+ *
+ * @param url The URL to open
+ * @returns Promise<void>
```

Comment:
Given that we are using typescript, this sort of annotation isn't really necessary here.

If you want to mark it as such then I would add it to the end of line 19

```
...(url: string): Promise<void> {
```